lib/tasks/import_csv.rake

```diff
@@ -0,0 +1,44 @@
+require 'csv'
```

Comment:
@brianzanti @dionew1 This is the rake task for importing from CSV data. Does this seem okay? I feel like it's loading the environment more than it needs to and I'm wondering if I should just combine all of the helper tasks and just have one task.

Reply:
I don't know if there's a good way to avoid loading the environment. I don' think it's a big deal though. It doesn't take a ton of time to load, and I'd prefer readability over the small performance boost. One thing you could consider is making another file in `lib` that contains a class that has all the helper methods in there, and your rake task can just import that class and call a method on it.